daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SValue.scala

```diff
@@ -174,6 +181,8 @@ object SValue {
 
   final case class SMap(value: HashMap[String, SValue]) extends SValue
 
+  final case class STNat(n: Int) extends SValue
```

Comment:
add a comment explaining what this is for and how and where it is used.

Reply:
done